openmetadata-service/src/main/java/org/openmetadata/service/OpenMetadataApplicationConfig.java

```diff
@@ -46,6 +47,9 @@ public class OpenMetadataApplicationConfig extends Configuration {
   @Valid
   private DataSourceFactory dataSourceFactory;
 
+  @JsonProperty("aws")
```

Comment:
@ArtyomyuS this should be optional and nullable. 
i.e users who don't pass this section should continue to run

Reply:
Ok, going to add it as optional

Reply:
@harshach isn't it by default optional btw, by specifying `@JsonProperty(value = "aws", required = false)` it looks to be redundant as the default is always `false` for the `required` parameter.